File: Java/BOJ/20125 쿠키의 신체 측정.java
import java.io.*;
import java.util.*;
import java.math.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());
        int heart_x = 0;
        int heart_y = 0;
        int back = 0;
        int arm_left = 0 ;
        int arm_right = 0 ;
        int leg_left = 0;
        int leg_right = 0;
        
        for (int i = 0 ; i < N; i++) {
        	String S = br.readLine();
        	for (int j = 0 ; j < N; j++) {
        		if (S.charAt(j) == '*') {
        			if (heart_x == 0 && heart_y == 0) {
        				heart_x = i+1;
        				heart_y = j;
        			}
        			else if (heart_x == i && heart_y > j) {
        				arm_left++;
        			}
        			else if (heart_x == i && heart_y < j) {
        				arm_right++;
        			}
        			else if (heart_x < i && heart_y == j) {
        				back++;
        			}
        			else if (heart_x + back < i && heart_y > j) {
        				leg_left++;
        			}
        			else if (heart_x + back < i && heart_y < j) {
        				leg_right++;
        			}
        		}
        	}
        }
        StringBuilder sb = new StringBuilder();
        sb.append(heart_x+1).append(" ");
        sb.append(heart_y+1).append("\n");
        sb.append(arm_left).append(" ");
        sb.append(arm_right).append(" ");
        sb.append(back).append(" ");
        sb.append(leg_left).append(" ");
        sb.append(leg_right).append(" ");
        System.out.println(sb);
    }
}

File: Python/BOJ/20125 쿠키의 신체 측정.py
import sys
N = int(sys.stdin.readline())
cookie = [list(sys.stdin.readline().strip()) for _ in range(N)]

heart_x, heart_y = 0, 0
arm_left, arm_right = 0, 0
back = 0
leg_left, leg_right = 0, 0
for i in range(N):
    for j in range(N):
        if cookie[i][j] == '*':
            if not heart_x and not heart_y:
                heart_x, heart_y = i+1, j
            elif heart_x == i and heart_y > j:
                arm_left += 1
            elif heart_x == i and heart_y < j:
                arm_right += 1
            elif heart_x < i and heart_y == j:
                back += 1
            elif heart_x + back < i and heart_y > j:
                leg_left += 1
            elif heart_x + back < i and heart_y < j:
                leg_right += 1

print(heart_x+1, heart_y+1)
print(arm_left,arm_right, back, leg_left,leg_right)
            